batch/batches/StorageUpdate/KAsyncStorageUpdate.php

```diff
@@ -308,7 +358,14 @@ public function handleUsage($percent, $partner, $systemPartnerConfiguration, $mo
 		}
 		else
 		{
-			$this->systemPartnerClientPlugin->systemPartner->updateConfiguration($partner->id, $systemPartnerConfiguration);
+			try
+			{
+				$this->systemPartnerClientPlugin->systemPartner->updateConfiguration($partner->id, $systemPartnerConfiguration);

```

Comment:
Same here - add small function partnerUpdateConfiguration - that will handle expection inside